src/localplayer.cpp

```diff
@@ -298,11 +299,24 @@ void LocalPlayer::move(f32 dtime, Environment *env, f32 pos_max_d,
 			new_sneak_node = p;
 		}
 
-		bool sneak_node_found = (min_distance_f < 100000.0*BS*0.9);
+		bool sneak_node_found = (min_distance_f < 100000.0 * BS * 0.9);
 
 		m_sneak_node = new_sneak_node;
 		m_sneak_node_exists = sneak_node_found;
 
+		if (sneak_node_found) {
+			f32 cb_max = 0;
+			MapNode n = map->getNodeNoEx(m_sneak_node);
+			std::vector<aabb3f> nodeboxes = n.getCollisionBoxes(nodemgr);
+			for (std::vector<aabb3f>::iterator i = nodeboxes.begin();
+					i != nodeboxes.end(); i++) {
```

Comment:
Can you pre-increment the iterator? post-increment will just create a needless  copy.